src/main/scala/org/broadinstitute/dsde/rawls/user/UserService.scala

```diff
@@ -73,12 +73,12 @@ class UserService(protected val userInfo: UserInfo, dataSource: SlickDataSource,
     case GetRefreshTokenDate => getRefreshTokenDate() pipeTo sender
 
     case CreateUser => createUser() pipeTo sender
-    case AdminGetUserStatus(userRef) => adminGetUserStatus(userRef) pipeTo sender
-    case UserGetUserStatus => userGetUserStatus() pipeTo sender
-    case EnableUser(userRef) => enableUser(userRef) pipeTo sender
-    case DisableUser(userRef) => disableUser(userRef) pipeTo sender
-    case ListUsers => listUsers pipeTo sender
-    case ImportUsers(rawlsUserInfoList) => importUsers(rawlsUserInfoList) pipeTo sender
+    case AdminGetUserStatus(userRef) => asAdmin { getUserStatus(userRef) } pipeTo sender
```

Comment:
im not sure i like asAdmin up here, since most other functions do withXX and other permission-y stuff down in the functions themselves.  It also opens us up to someone reusing a function below and forgetting the asAdmin